### PR TITLE
Do not hard code API host of push notification service

### PR DESCRIPTION
**Pull request by jasonpet**
fix for issue #85

### CODE REVIEW

**Comment by jasonpet** [comment]
@csantanapr please review

File: packages/actions/sendMessage.js [hunk 1]
```diff
@@ -426,10 +427,22 @@ function main(params) {
 
   var bodyData = JSON.stringify(sendMessage);
   var request = require('request');
+  var apiHost;
+  if (params.apiHost) {
+    apiHost = params.apiHost;
+  }
+  else if (params.admin_url) {
+    var adminURL = require('url').parse(`https:${params.admin_url}`);
```
Comment (csantanapr) [inline packages/actions/sendMessage.js]: Need to check if admin url starts with http first
Today it starts with // but in future might be corrected with https://

Comment (jasonpet) [reply packages/actions/sendMessage.js]: good idea

**Review by csantanapr — APPROVED** [review]
Thanks @jasonpet for the contribution

**Pull request merged**